components/data-area.tsx

```diff
@@ -82,6 +84,25 @@ const DataArea = (props: any) => {
         // getGithubData(githubUserName);
     }, [originalSearchVal]);
 
+    const renderData = (currentFilter: null | string) => {
+        switch (currentFilter) {
+            case null:
+                return (
+                    <>
+                        <GithubArea {...searchVal} />
+                        <CodePenArea {...searchVal} />
+                        <LeetCodeArea {...searchVal} />
+                    </>
+                );
+            case 'LEETCODE':
```

Comment:
@Zishan3165 better to use enum which would be beneficial later in other pages too.